uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -44,7 +45,9 @@ define([
       this.postContactSelection = false; // flag to track if user is selected for enabling UI
       this.requestStatuses = {};
       this.statusBeforeEdit = {};
+      this.supportedFileTypes = '';
       this.today = Date.now();
+      this.userRemovedFile = false; // flag to track if user has removed file
```

Comment:
ps refer to function where this property is set 
```
      /**
       * Removes attachment. Also, sets the flag to let UI know that a removal has happened.
       * @param {String} fromWhere source from where deletion is being made like
       * from files array or fileuploader's queue
       * @param {Object} file
       */
      this.removeAttachment = function (fromWhere, file) {
        if (fromWhere === 'files') {
          this.request.deleteAttachment(file);
        } else if (fromWhere === 'queue') {
          this.request.fileUploader.removeFromQueue(file);
        }
        this.userRemovedFile = true;
      };
```
it is far more complex to get notifications from them whether deletion happened and tracking it in a separate function than this flag, if you have alternative ps suggest